src/panes/index.jsx

```diff
@@ -3,12 +3,14 @@ import ManualInputPane from './ManualInput'
 import TestComponent from './TestComponent'
 import SplitPane from './SplitPane'
 import StatusPane from './StatusPane'
+import SciencePane from './SciencePane'
```

Comment:
We can remove these references to SciencePane now that it lives in the ManualInput pane